.github/workflows/windows.yml

```diff
@@ -26,3 +26,6 @@ jobs:
       - name: Run Windows Unit Tests
         run: |
           go test -v -race ./internal/...
+      - name: Run Windows Integration Tests
+        run: |
+          go test -v -race ./integrationtests/...
```

Comment:
The integration tests expect csi-proxy binary to be built and running with the named pipes for the API groups listening. Otherwise, the tests would fail when trying to connect to the named pipes. Please build and start csi-proxy before executing the tests.

Reply:
@ddebroy do you have guideline about how to do this?  especially this: `running with the named pipes for the API groups listening`  @mayankshah1607 could help while need your guide first, thanks.

Reply:
There are a few projects depending on this csi-proxy, and if there is an example solution about how to run unit test on csi-proxy, that would be great.

Reply:
@andyzhangx sorry about the delay in responding. The unit tests for csi-proxy are rather minimal as there isn't a lot of internal logic. I think the focus of this PR around integration tests would be ideal to enable. The only piece missing here is building and running csi-proxy before invoking `go test -v -race ./integrationtests/...` . 

csi-proxy can be built and run using `go build -a -o _output/csi-proxy.exe ./cmd/csi-proxy` as the build action is already doing followed by starting csi-proxy.exe by simply starting the executable: `csi-proxy.exe` . Note that `csi-proxy.exe` will run indefinitely, so if github action has a way to start it as a background exe that will be great. Alternative would be to register and start `csi-proxy.exe` in service mode: 

```
sc create csiproxy binPath= "C:\csiproxy\csi-proxy.exe --windows-service"
net start csiproxy
```

Reply:
We're just running the CSI proxy as a background process in the csi-driver-smb repo - https://github.com/kubernetes-csi/csi-driver-smb/blob/master/.github/workflows/windows.yml#L28

Maybe we could do the same here as well?

Reply:
@mayankshah1607 pls go ahead with another PR, I think there could be a lot more work to do besides github action config change. Thanks.